From f1da8a190e8f80a71b169ed5e5d7311fd53c9046 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sun, 11 Feb 2018 20:20:51 +0800
Subject: [PATCH] sound/soc/codecs/rk817_codec: fix compile warning

sound/soc/codecs/rk817_codec.c:405:14: warning: duplicate const
declaration specifier [-Wduplicate-decl-specifier]

Change-Id: I572b1778f29a48b0756ac60488ce2311cc372f8b
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 sound/soc/codecs/rk817_codec.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sound/soc/codecs/rk817_codec.c b/sound/soc/codecs/rk817_codec.c
index c1f862966ab8..2ec16c291c5a 100644
--- a/sound/soc/codecs/rk817_codec.c
+++ b/sound/soc/codecs/rk817_codec.c
@@ -402,16 +402,16 @@ static const char * const rk817_call_path_mode[] = {
 
 static const char * const rk817_modem_input_mode[] = {"OFF", "ON"};
 
-static const SOC_ENUM_SINGLE_DECL(rk817_playback_path_type,
+static SOC_ENUM_SINGLE_DECL(rk817_playback_path_type,
 	0, 0, rk817_playback_path_mode);
 
-static const SOC_ENUM_SINGLE_DECL(rk817_capture_path_type,
+static SOC_ENUM_SINGLE_DECL(rk817_capture_path_type,
 	0, 0, rk817_capture_path_mode);
 
-static const SOC_ENUM_SINGLE_DECL(rk817_call_path_type,
+static SOC_ENUM_SINGLE_DECL(rk817_call_path_type,
 	0, 0, rk817_call_path_mode);
 
-static const SOC_ENUM_SINGLE_DECL(rk817_modem_input_type,
+static SOC_ENUM_SINGLE_DECL(rk817_modem_input_type,
 	0, 0, rk817_modem_input_mode);
 
 static int rk817_playback_path_get(struct snd_kcontrol *kcontrol,
-- 
2.35.3

